src/components/home/Landing.jsx

```diff
@@ -1,7 +1,32 @@
+import Image from "next/image";
+import image1 from "@/public/home/landing.webp";
+import image2 from "@/public/waveBottom.webp";
+import Button from "../Button";
+
 const Landing = () => {
   return (
-    <div className="">
-      <p>Landing</p>
+    <div className="w-screen relative -translate-y-5">
+      <Image src={image1} alt="Background image 1" className="w-screen" />
+      <div className="w-[50%] h-full bg-swim-blue-200/60 top-[0%] absolute"></div>
```

Comment:
```suggestion
      <div className="w-[50%] h-full bg-swim-blue-200/60 top-[0%] absolute">
      [TITLE AND DESCCRIPTION]
      </div>
```

Move the title and description and put them inside of this divider tag. We want them to be a part of the blue tint group. You can center the text using flex and justify.

https://tailwindcss.com/docs/justify-content